docs/axes/cartesian/time.md

```diff
@@ -94,4 +94,4 @@ var chart = new Chart(ctx, {
 ## Parser
 If this property is defined as a string, it is interpreted as a custom format to be used by moment to parse the date. 
 
-If this is a function, it must return a moment.js object given the appropriate data value.
\ No newline at end of file
+If this is a function, it must return a moment.js object given the appropriate data value.
```

Comment:
I guess my editor may have inserted a newline at the end of file. I think this is a good practice.